scripts/legislation/africa/south_africa_english.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^I. In this Act/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
```

Comment:
Due to the `^` character in the regular expression, the expression will only match numbers and letters in parentheses at the beginning of a line. Is that intended?

Reply:
No, thanks for the catch. I have amended the rest of the script to adjust for this change.